llama-index-integrations/embeddings/llama-index-embeddings-cohere/llama_index/embeddings/cohere/base.py

```diff
@@ -261,6 +281,38 @@ async def _aembed(self, texts: List[str], input_type: str) -> List[List[float]]:
             ).embeddings
         return getattr(result, self.embedding_type, None)
 
+    def _embed_image(self, image_path: ImageType, input_type: str) -> List[float]:
+        """Embed images using Cohere."""
+        if self.model_name not in V3_MODELS:
+            raise ValueError(
+                f"{self.model_name} is not a valid multi-modal embedding model. Supported models are {V3_MODELS}"
+            )
+        client = self._get_client()
+        processed_image = self._image_to_base64_data_url(image_path)
```

Comment:
If the image path is already a bytes object of the image, we might to to change how we convert that?